knowledge-base/shivammathur/setup-php/action-security.yml

```diff
@@ -0,0 +1,116 @@
+name: 'setup-php'
+github-token:
+  environment-variable-name: COMPOSER_TOKEN
+  permissions:
+    contents: read
+    contents-reason: to access matching refs GitHub API, clone the repos, and get release assets.
+outbound-endpoints:
+  api.github.com:443:
+    port: 443
+    reason: to call GitHub API
+  api.launchpad.net:443:
+    port: 443
+    reason: to call launchpad API
+  archive.ubuntu.com:443:
+    port: 443
+    reason: to download packages from ubuntu archives
+  archive.ubuntu.com:80:
```

Comment:
The port needs to be specified in the port key. This applies to all entries

Reply:
Fixed